node/src/actors/chain_manager/mod.rs

```diff
@@ -193,6 +196,18 @@ pub struct ChainManager {
     /// Enable superblock creation
     create_superblocks: bool,
 }
+/// Struct thar resume a block candidate
```

Comment:
```suggestion
/// Wrapper around a block candidate that contains additional metadata regarding needed chain state mutations in case the candidate gets consolidated.
```